aries-site/src/layouts/content/WCAGRuleSummary.js

```diff
@@ -24,6 +20,10 @@ const STATUS_MAP = {
     label: 'Conditional',
     icon: <CircleInformation alt="" />,
   },
+  'AAA failed': {
+    label: 'AAA Failed',
+    icon: <StatusWarningSmall alt="" color="status-warning" />,
+  },
 };
 
 const StatusLabel = ({ icon, label }) => (
```

Comment:
Trying to get the numerical values to right-aligned. Can we try adding a `justify="between"` on the Box? Not sure `textAlign` will help in this circumstance, but it might be another option.

Reply:
fixed